app/views/posts/show.html.erb

```diff
@@ -1 +1,36 @@
-A post details with a list of comments
\ No newline at end of file
+<div class="user-wrapper">
+    <div class="user-container">
+        <div class="user-list">
+            <span class="photo">
+                <%= image_tag(@post.user.photo, size: "100x100" ) %>
+            </span>
+            <span class="user-info">
+                <span>
+                    <%= link_to @post.user.name, user_path(user_id: @post.user.id, format: :html) %>
+                </span>
+                <span>
+                    number of posts: <%= @post.user.posts_counter ? @post.user.posts_counter : 0 %>
+                </span>
+            </span>
+        </div>
+        <ul class="posts">
+            <li>
+                <span>
+                    <%= link_to "Post #: #{@post.id}" , user_post_path(user_id: @post.user.id, post_id: @post.id) %>
+                    by <%=@post.user.name%>
+                </span>
+                <span>
+                    <%= @post.text %>
+                </span>
+                <span>comments: <%= @post.comments.count %>&nbsp; likes: <%= @post.likes.count %></span>
+            </li>
+        </ul>
+        <ul class="comments">
+            <%@post.comments.each do |comment|%>
+                <li>
+                    <%=comment.user.name%> : <%=comment.text%>
```

Comment:
- [Optional] good job on implementing this recent comments method but you should also use it here 
![image](https://user-images.githubusercontent.com/74744787/214711943-ca16a06d-6457-4a9e-84ec-51930c2cef0e.png)
instead of writing the whole code again